apps/dialtone-documentation/docs/design/colors/themes/index.md

```diff
@@ -0,0 +1,209 @@
+  </template>
+</dialtone-usage>
+
+<div class="d-m32"></div>
+
+### CSS Implementation
+
+```css
+/* ✅ Correct - Uses theme-aware semantic tokens */
+.my-component {
+  background-color: var(--dt-theme-surface-primary);
+  color: var(--dt-theme-foreground-primary);
+  border-color: var(--dt-theme-border-default);
+}
+
+/* ❌ Incorrect - Uses base tokens that don't adapt to themes */
+.my-component {
+  background-color: var(--dt-color-black-100);
+  color: var(--dt-color-black-900);
+  border-color: var(--dt-color-black-300);
+}
+```
+
+## Customize Theming
+
+For advanced use cases, you can create custom themes by defining your own token mappings while maintaining the semantic structure.
+
+<div class="d-bgc-brand d-bar8"><svg-loader class="d-fl1" name="custom-themes" /></div>
+
+<div class="d-m32"></div>
+
+### Creating Custom Themes
+
+1. **Define your color palette**: Start with base color tokens that align with your brand
+2. **Map semantic meanings**: Assign base colors to semantic roles (primary, secondary, etc.)
+3. **Test accessibility**: Ensure contrast ratios meet WCAG guidelines
+4. **Validate consistency**: Check that your theme works across all component states
+
+### Custom Theme Example
+
+```css
+/* Custom brand theme */
+:root[data-theme="custom-brand"] {
+  --dt-theme-surface-primary: var(--dt-color-blue-50);
+  --dt-theme-surface-secondary: var(--dt-color-blue-100);
+  --dt-theme-foreground-primary: var(--dt-color-blue-900);
+  --dt-theme-foreground-secondary: var(--dt-color-blue-700);
+  --dt-theme-accent-default: var(--dt-color-gold-500);
+  --dt-theme-accent-hover: var(--dt-color-gold-600);
+}
+```
+
+## Apply Themes
+
+Themes are applied at the application level using data attributes or CSS classes, allowing for dynamic switching and scoped theme application.
+
+<div class="d-bgc-brand d-bar8"><svg-loader class="d-fl1" name="apply-themes" /></div>
+
+<div class="d-m32"></div>
+
+### Theme Application Methods
+
+#### Data Attribute Method (Recommended)
+
+```html
+<html data-theme="dark">
+  <!-- All child elements inherit dark theme -->
+</html>
+```
+
+#### CSS Class Method
+
+```html
+<div class="dt-theme-light">
+  <!-- Scoped light theme application -->
+</div>
+```
+
+### JavaScript Theme Switching
+
+```javascript
+// Switch to dark theme
+document.documentElement.setAttribute('data-theme', 'dark');
+
+// Switch to light theme  
+document.documentElement.setAttribute('data-theme', 'light');
+
+// Switch to custom theme
+document.documentElement.setAttribute('data-theme', 'custom-brand');
+```
+
+### Responsive Theme Application
+
+Themes can be applied responsively based on user preferences or system settings:
+
+```css
+/* Respect user's system preference */
+@media (prefers-color-scheme: dark) {
+  :root:not([data-theme]) {
+    /* Apply dark theme tokens */
+  }
+}
+
+@media (prefers-color-scheme: light) {
+  :root:not([data-theme]) {
+    /* Apply light theme tokens */
+  }
+}
+```
```

Comment:
This isn't quite right, and should be authored by a DT engineer. It might already be covered elsewhere. Ask them.